docs/src/main/sphinx/connector/mariadb.rst

```diff
@@ -71,41 +71,199 @@ that catalog name instead of ``mariadb`` in the above examples.
-``double``                          ``double``
-``decimal(p,s)``                    ``decimal(p,s)``
-``char(n)``                         ``char(n)``
-``tinytext``                        ``varchar(255)``
-``text``                            ``varchar(65535)``
-``mediumtext``                      ``varchar(16777215)``
-``longtext``                        ``varchar``
-``varchar(n)``                      ``varchar(n)``
-``tinyblob``                        ``varbinary``
-``blob``                            ``varbinary``
-``mediumblob``                      ``varbinary``
-``longblob``                        ``varbinary``
-``varbinary(n)``                    ``varbinary``
-``date``                            ``date``
-``time(n)``                         ``time(n)``
-``timestamp(n)``                    ``timestamp(n)``                MariaDB stores the current timestamp by default.
-                                                                    Please enable `explicit_defaults_for_timestamp <https://mariadb.com/docs/reference/mdb/system-variables/explicit_defaults_for_timestamp/>`_
-                                                                    to avoid implicit default values.
-==================================  =============================== =============================================================================================================
+Because Trino and MariaDB each support types that the other does not, this
+connector modifies some types when reading or writing data. Data types may not
+map the same way in both directions between Trino and the data source. Refer to
+the following sections for type mapping in each direction.
+
+MariaDB type to Trino type mapping
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector maps MariaDB types to the corresponding Trino types according
+to the following table:
+
+.. list-table:: MariaDB type to Trino type mapping
+  :widths: 30, 30, 50
+  :header-rows: 1
+
+  * - MariaDB type
+    - Trino type
+    - Notes
+  * - ``boolean``
+    - ``tinyint``
+    -
+  * - ``tinyint``
+    - ``tinyint``
+    -
+  * - ``smallint``
+    - ``smallint``
+    -
+  * - ``int``
+    - ``integer``
+    -
+  * - ``bigint``
+    - ``bigint``
+    -
+  * - ``tinyint unsigned``
+    - ``smallint``
+    -
+  * - ``smallint unsigned``
+    - ``integer``
+    -
+  * - ``mediumint unsigned``
+    - ``integer``
+    -
+  * - ``integer unsigned``
+    - ``bigint``
+    -
+  * - ``bigint unsigned``
+    - ``decimal(20,0)``
+    -
+  * - ``float``
+    - ``real``
+    -
+  * - ``double``
+    - ``double``
+    -
+  * - ``decimal(p,s)``
+    - ``decimal(p,s)``
+    -
+  * - ``char(n)``
+    - ``char(n)``
+    -
+  * - ``tinytext``
+    - ``varchar(255)``
+    -
+  * - ``text``
+    - ``varchar(65535)``
+    -
+  * - ``mediumtext``
+    - ``varchar(16777215)``
+    -
+  * - ``longtext``
+    - ``varchar``
+    -
+  * - ``varchar(n)``
+    - ``varchar(n)``
+    -
+  * - ``tinyblob``
+    - ``varbinary``
+    -
```

Comment:
`mediumblob` is missing that exists in the original type mapping.

Reply:
Added back in